src/pyuvdata/uvbeam/uvbeam.py

```diff
@@ -60,6 +60,10 @@ class UVBeam(UVBase):
             "description": "scipy RectBivariate spline interpolation",
             "func": "_interp_az_za_rect_spline",
```

Comment:
We discussed maybe changing the name of this option (with deprecation!) to include something related to RectBivariateSpline. I think maybe it's best to push that to another PR, just wanted to mention it here.

Reply:
moved this to an issue: #1473